doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -14,38 +14,58 @@
+* The NFVI KPIs are reviewed and compared with Golden KPIs to determine if the certification badge is to be provided to NFVI or not.
 
 <a name="2.3"></a>
 ## 2.3 Certification Strategy & Vehicle
-
-Define and illustrate the certification strategy and vehicle for NFVI..
+In order to begin the certification process, NFVI needs to be validated and expected to be in a required state. This state would be determined by running tests as described in Reference Implementation. Once the target NFVI passes these tests, it would become a candidate for NFVI certification. If NFVI fails the tests, it will not be move to next workflow for certification.
 
 <a name="2.4"></a>
 ## 2.4 Profiles Reference 
-
-Define the profiles used, why they're important, and what is being certified..
+Different vendors have different types of VNFs to serve different use-cases. A VNF like Broadband Network Gateway (BNG) would require high netorking throughout whereas a VNF like Mobility Management Entity (MME) woud require high computing performance. As such, BNG would require high KPI values for network throughput and MME would require high CPU performance KPIs like Index Score, Instructions Per Second (IPS) etc. The target NFVI to cater these needs woud have different characteristics. Depending on VNF's requirements, the NFVI can be categorised into below profiles:
+* Basic (B) profile for standard computing
+* Compute intensive (C) profile where predictable computing performance is expected and
+* Network intensive (N) profile offerring low latency and high networking throughout
+Similarly, different NFVI vendors may specialise in different hardware profiles and some may specialise in both VNFs and NFVI. To cater to different needs from multiple NFVI vendors, CNTT allows different types of NFVI certification based on their types of [profile](../../../ref_model/chapters/chapter02.md#2.3)
+ * Certify Vendor NFVI Hardware solution: This allows for certification of only NFVI.
+ * Certify Vendor NFVI Hardware and Software Solution: This allows for certification for NFVI running a particular VNF.
 
 <a name="2.5"></a>
 ## 2.5 Compliance, Verification, and Certification
-
-Define the framework and process for NFVI certification..
+The below set of steps define the compliance, verification and certification process for NFVI
+* Based on VNF's requirements, NFVI profile is selected - B, C, N
+* NFVI readiness is checked for certification.
+* The test VNFs are on-boarded using automation scripts on the NFVI.
+* VNF on-boarding is validated by running functional tests to ensure that the on-boarding is successful.
+* VNF performance tests are executed and NFVI KPIs are recorded during the tests.
+* KPI comparison is run to compare NFVI KPIs with Golden KPIs, which serve as a reference for NFVI certification.
+* If NFVI KPIs meet Golden KPIs, NFVI is certified and granted a certification badge.
+* If NFVI KPIs do not meet Golden KPIs, no certification is provided.
 
```

Comment:
Add image.
![NFVI_ref_profiles_impementations](https://user-images.githubusercontent.com/5643130/71145652-f0d6ea80-21e7-11ea-92bc-6a27c22d650f.jpg)

Reply:
Added.

Reply:
Let me know if we can resolve this.

Reply:
same here - @msidana - Manik, I could **not** see the image.  Did you upload the image to the figure directory?  CNTT/doc/ref_cert/lfn/figures/ .  Besides the image, the text looks good.

Reply:
@mf4716 Yes, even I can't see them. All figures are uploaded to ```doc/ref_cert/lfn/figures/``` directory and are showing up in "Files Changed" tab of this PR.

Reply:
Okay.  Let's close/resolve and address any issues after merge.  I'm fairly confident it will work once the images are successfully updated (merged) in the Figures directory.  Resolving this thread.